tests/test-sources/plugins/languages/treesitter/treesitter.nix

```diff
@@ -120,4 +120,39 @@
       ];
     };
   };
+
+  combine-plugins = {
```

Comment:
This test seems to fail on darwin (https://github.com/nix-community/nixvim/pull/1886#issuecomment-2247402248)

@traxys is `tests.dontRun = pkgs.stdenv.isDarwin` enough or should the relevant `enable` options be set to that instead?

Reply:
No, we need to disable the `enable` I believe

Reply:
(it's always annoying to hit cases like this --')

Reply:
What path do you want?

* It should work when splitting the tests (https://buildbot.nix-community.org/#/builders/2798/builds/1, https://buildbot.nix-community.org/#/builders/1856/builds/19).
* It probably can work when reducing the number of parsers (instead of all, select only a number of them).
* Disable test on darwin.

Reply:
I'd say splitting the tests could be the best option, followed by reducing the parsers and disable the test as a last resort

Reply:
> I'd say splitting the tests could be the best option, followed by reducing the parsers and disable the test as a last resort

OK, I'll squash with treesitter commit when it finished if there won't be another opinions.